t/t2233-job-info-update.t

```diff
@@ -69,7 +69,6 @@ test_expect_success 'job-info: update lookup with no update events works (job in
 test_expect_success 'job-info: update lookup with update events works (job active)' '
 	jobid=$(flux submit --wait-event=start sleep 300) &&
 	update1=$(expiration_add $jobid 100) &&
-	flux job eventlog $jobid &&
```

Comment:
Debugging in tests is useful when the test fails, so you can actually see what was happening. If anything this should be under `test_debug`, but functionally it is no different. If the output isn't harming anything, maybe just leave it in?